document available host preflights types (#656)
replicatedhq/kurl.sh: src/markdown-pages/create-installer/host-preflights/tcp-port.md

---
path: "/docs/create-installer/host-preflights/tcp-port-status"
date: "2022-01-13"
weight: 35
linktitle: "TCP Port Status"
title: "TCP Port Status"
---
 

> The TCP port status host preflight check can be usee to detect the status of a TCP port to check whether it's open and available.

Reviewer: ```suggestion
The TCP port status host preflight check is used to detect the status of a TCP port to check whether it's open and available.
```
